apps/dashboard/package.json

```diff
@@ -106,6 +106,7 @@
     "typescript": "5.6.2",
     "typescript-eslint": "^8.0.1",
     "vite": "^5.4.1",
+    "vite-plugin-ejs": "^1.7.0",
```

Comment:
the Vite EJS plugin that allows the use of conditionals in the HTML template; by default, it's not supported by Vite :/